### DISCUSSION TITLE
Join Us at NVIDIA's GTC for an In-depth Session on the CUDA C++ Core Libraries!

### DISCUSSION THREAD

**Discussion by jrhemstad** [NVIDIA/cccl#1525] [Announcements]
Hello CUDA C++ enthusiasts!

I'm thrilled to announce that @gevtushenko and I will be presenting at the upcoming NVIDIA GTC conference. Our session, ["Mastering CUDA C++: Modern Best Practices with the CUDA C++ Core Libraries" [S62175]](https://www.nvidia.com/gtc/session-catalog/?#/session/1695315322330001eIye), is tailored for developers like you who are passionate about pushing the boundaries of CUDA C++.

In this session, we'll dive deep into the art of writing high-performance, safe, and adaptable CUDA C++ code. We'll explore how the CUDA C++ Core Libraries (Thrust, CUB, and libcudacxx) can be powerful allies in your development toolkit. Additionally, we'll discuss how to use CMake to streamline building of your CUDA C++ projects.

Here are the details:
Date: Wednesday, March 20
Time: 8:00 AM - 8:50 AM PDT
Location: SJCC LL20C (LL)
Registration: [Click here to register](https://www.nvidia.com/gtc/session-catalog/?#/session/1695315322330001eIye)

This is also a fantastic opportunity for you to come meet and talk with us. We have a [GTC Meetup](https://gtc24.braindate.com/topics/120844/) scheduled where we're looking forward to engaging with the community, answering your questions, and discussing the future of CUDA C++ development.

Looking forward to seeing many of you there!

**Comment by lee-vincent** [comment]
@jrhemstad was this session recorded? It is not available on-demand. Thank you!
